docs/app/Examples/elements/Header/Content/HeaderIconExample.js

```diff
@@ -0,0 +1,15 @@
+import React, {Component} from 'react';
+import {Header} from 'stardust';
+
+export default class HeaderIconExample extends Component {
+  render() {
+    const plug = <i className='plug icon' />;
+    return (
+      <Header.H2 icon={plug}>
+        <div className='content'>
```

Comment:
Let's handle this `content` wrapper in the component logic.  If there is an icon prop, we can wrap `this.props.children` in the `content` div.